packages/clerk-js/src/ui/elements/DevModeNotice.tsx

```diff
@@ -0,0 +1,54 @@
+import type { ThemableCssProp } from 'ui/styledSystem';
+
+import { useEnvironment } from '../contexts';
+import { Box, Text } from '../customizables';
+
+export const DevModeOverlay = () => {
+  const { isDevelopmentOrStaging } = useEnvironment();
+
+  if (!isDevelopmentOrStaging()) {
+    return null;
+  }
+
+  return (
+    <Box
+      sx={t => ({
+        userSelect: 'none',
+        pointerEvents: 'none',
+        width: '100%',
```

Comment:
You can remove this width, height, top, and left and replace with just `inset: 0` 👍